ocaml/xapi/xapi_vlan.ml

```diff
@@ -31,14 +31,15 @@ let create_internal ~__context ~host ~tagged_PIF ~tag ~network ~device =
   (* Copy the MTU and metrics from the base PIF *)
   let mTU = Db.PIF.get_MTU ~__context ~self:tagged_PIF in
   let metrics = Db.PIF.get_metrics ~__context ~self:tagged_PIF in
+  let pci = Db.PIF.get_PCI ~__context ~self:tagged_PIF in
   Db.PIF.create ~__context ~ref:untagged_PIF ~uuid:(Uuid.to_string (Uuid.make_uuid ()))
     ~device ~device_name:device ~network ~host ~mAC:vlan_mac ~mTU ~vLAN:tag ~metrics
     ~physical:false ~currently_attached:false ~igmp_snooping_status:`unknown
     ~ip_configuration_mode:`None ~iP:"" ~netmask:"" ~gateway:"" ~dNS:"" ~bond_slave_of:Ref.null
     ~vLAN_master_of:vlan ~management:false
     ~other_config:[] ~disallow_unplug:false
     ~ipv6_configuration_mode:`None ~iPv6:[""] ~ipv6_gateway:"" ~primary_address_type:`IPv4 ~managed:true
-    ~properties:[] ~capabilities:[];
+    ~properties:[] ~capabilities:[] ~pCI:pci;
```

Comment:
I think that we should leave it NULL for any non-physical PIF (as for the bond master and tunnel access PIF above).

Reply:
yes